deployment/deploy/7-PriceFeedRevertOnStalePrice.ts

```diff
@@ -0,0 +1,20 @@
+import { DeployFunction } from "hardhat-deploy/types";
+const path = require("path");
+
+const func: DeployFunction = async (hre) => {
```

Comment:
as fairly noted by @devtective the deployment should replace the logic of the PriceFeed proxy.  
for the RSK mainnet pls print a note to create a SIP to complete the upgrade with the new logic address.

Reply:
Not relevant anymore with the latest changes in this [commit](https://github.com/DistributedCollective/zero-contracts/pull/6/commits/d89d7837edf44a897517a27d13825f8679133fec)

Reply:
Not relevant anymore with the latest changes in this [commit](https://github.com/DistributedCollective/zero-contracts/pull/6/commits/d89d7837edf44a897517a27d13825f8679133fec)